src/use_case/display_favourite_recipe/DisplayFavouriteOutputBoundary.java

```diff
@@ -0,0 +1,12 @@
+package use_case.display_favourite_recipe;
+
+import entity.Recipe;
+import java.util.List;
+public interface DisplayFavouriteOutputBoundary {
+    /**
+     * Prepares the success view for the user.
+     * @param recipes the recipes to display.
+     */
+    void prepareSuccessView(List<Recipe> recipes);
```

Comment:
The success view should probably take a `DisplayFavouriteOutputData` object, instead of a `List<Recipe>`.